packages/x-date-pickers/src/internals/components/PickersViewLayout/PickersViewLayout.tsx

```diff
@@ -14,33 +15,67 @@ export const PickersViewLayoutRoot = styled('div', {
   name: 'MuiPickersViewLayout',
   slot: 'Root',
   overridesResolver: (props, styles) => styles.root,
-})({
-  display: 'flex',
-  flexDirection: 'column',
-});
+})(({ theme }) => ({
+  display: 'grid',
+  gridAutoColumns: 'max-content auto max-content',
+  gridAutoRows: 'max-content auto max-content',
+  [`&.${pickersViewLayoutClasses.landscape} .${pickersViewLayoutClasses.toolbar}`]: {
```

Comment:
👍 I do feel that, even though it makes even the default styles a bit more terse, in the end it provides a less cumbersome way for end users to override the layout. 🤔 
```tsx
componentsProps={{
  layout: {
    sx: {
      [`&.${pickersViewLayoutClasses.landscape}`]: {
        [`.${pickersViewLayoutClasses.content}`]: {
          gridColumn: 1,
        },
        [`.${pickersViewLayoutClasses.toolbar}`]: {
          gridColumn: 2,
        },
      }
    }
  }
}}
```

Reply:
I replace it with 
```jsx
...(ownerState.isLandscape && {
    [`& .${pickersViewLayoutClasses.toolbar}`]: {
      gridColumn: theme.direction === 'rtl' ? '3' : '1',
      gridRow: '1 / 3',
    },
  }),
```

Such that the CSS Selector only use `&` and `.toolbar` classes, so developpers can usn `&.isLandscape .toolbar` to be more specific and override it.

Reply:
Sure, that seems to also work. 🤔 
Just don't forget the "It also uncovers an issue of not correctly re-exporting the `pickersViewLayoutClasses`" point to avoid following import:
```ts
import { pickersViewLayoutClasses } from '@mui/x-date-pickers/internals/components/PickersViewLayout/pickersViewLayoutClasses';
```

Reply:
Are you sure moving this component to the first level export is the best course of action? 🤔 
I.e.: `PickersPopper` is somewhat on the same level of importance but is exported and placed in `internals` 🤔 
Your reasoning is probably the fact that its classes might be used quite frequently and thus exporting them from `/internals` is not a nice DX?